<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UndergroundFloor</class>
 <widget class="QWidget" name="UndergroundFloor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/assists/All_Materials/Underground_Floor.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget::item:selected {
  background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                              stop:0 #a188d3, stop:1 #8bcdff);

}
QTableWidget
{
   background-color:transparent;
border:none;
gridline-color:transparent;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="rowCount">
    <number>37</number>
   </property>
   <property name="columnCount">
    <number>76</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>1</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>1</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QPushButton" name="storage1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>491</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
color: white;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="room1">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>428</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
color: white;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="room2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
color: white;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="class6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>141</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
text {
    color: white;
    font-size: 12px;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="room3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
color: white;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="class5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>171</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
text {
    color: white;
    font-size: 12px;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="room4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>450</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
color: white;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="csSystem">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>130</y>
     <width>161</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
text {
    color: white;
    font-size: 12px;}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="room5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>440</y>
     <width>161</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
color: white;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="electronics">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>570</y>
     <width>491</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
text {
    color: white;
    font-size: 12px;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="room6">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>300</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
color: white;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="room7">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>230</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
color: white;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="room8">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>130</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
color: white;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="storage2">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>440</y>
     <width>101</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>-1</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
color: white;
font-size: 12px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="equipMaintenance">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>360</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Rounded MT Bold</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: transparent;
border:none;
background-color: transparent;
text {
    color: white;
    font-size: 12px;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="menuButton">
   <property name="geometry">
    <rect>
     <x>1170</x>
     <y>50</y>
     <width>61</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Materials.qrc">
     <normaloff>:/assists/All_Materials/Tasks_right.png</normaloff>:/assists/All_Materials/Tasks_right.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>50</width>
     <height>25</height>
    </size>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>10</y>
     <width>101</width>
     <height>162</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Undergrounds</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="menuButton_3">
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Materials.qrc">
        <normaloff>:/assists/All_Materials/Tasks_right.png</normaloff>:/assists/All_Materials/Tasks_right.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="nextButton_2">
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
border: none;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Materials.qrc">
        <normaloff>:/assists/All_Materials/Right arrow.png</normaloff>:/assists/All_Materials/Right arrow.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="previousButton_2">
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Materials.qrc">
        <normaloff>:/assists/All_Materials/Left_arrow.png</normaloff>:/assists/All_Materials/Left_arrow.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backButton_2">
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="Materials.qrc">
        <normaloff>:/assists/All_Materials/Undo.png</normaloff>:/assists/All_Materials/Undo.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Materials.qrc"/>
 </resources>
 <connections/>
</ui>
